app/src/test/java/br/com/catbag/gifreduxsample/reducers/middlewares/RestMiddlewareTest.java

```diff
@@ -108,8 +107,8 @@ public void whenInterceptListFetching() throws Exception {
     public void whenInterceptListFetchingAndDontHasMoreGifs() throws Exception {
         AppState state = ImmutableAppState.builder().hasMoreGifs(false).build();
         DataManager dataManager = mockDataManager();
-        RestMiddleware middleware = new RestMiddleware(RuntimeEnvironment.application,
-                dataManager, null);
+        RestMiddleware middleware = new RestMiddleware(application,
+                mockDataManager(), null);
```

Comment:
Sorry....i see your comment but forget to reply. You`re right on your point. I don`t see the usage of `dataManager` below. Next time i will pay more attention on my PR revision. 👎 for me...